### ISSUE TITLE
Switching buffers loses centering

### ISSUE THREAD

**Issue by peter-subsplash** [arnamak/stay-centered.nvim#5]
When pressing `^` to switch buffers, the centering is lost if one of the buffers is scrolled to the bottom of the page. This is a bit jarring.

Would it be possible to make this run on BufEnter or in some way make it re-center on switching buffers?

**Comment by arnamak** [comment]
@peter-subsplash potentially. tbh I’ve never really been a buffer person, I can take a crack at it when I’ve got some time but could you ELI5 the scenario and give me an example of how I can repro?

AFAIK all `^` does is take you to the beginning of a line in visual mode, so that probably gives you an idea of how unfamiliar I am with buffers.

**Comment by peter-subsplash** [comment]
Oh, I'm sorry! I meant CTRL-6 (or SHIFT-CTRL-6).

**Comment by peter-subsplash** [comment]
I'm using your plugin in Lunarvim, so it may be something else is complicating things. I can try to make a minimum-repro-setup using plain Neovim or something.

**Comment by arnamak** [comment]
@peter-subsplash any luck? if it's still a problem, I have some time to take a look soon.

**Comment by peter-subsplash** [comment]
I just updated and it's working! Yay!
This apparently fixed it 89c75f62b0607928bd34313d48516a7ec0a87cc1

Thanks!
